docs/pages/en-us/Architecture.md

```diff
@@ -13,26 +13,48 @@ banner:
 
 # Architecture
 
-The Audit library logs audit event metadata, belonging to tenant applications, into Elasticsearch.
+The Audit library logs audit event metadata, belonging to tenant applications, into Elasticsearch directly or via the Audit Web Service HTTP REST API.
 
-In order to use Auditing with an application, you must first specify the audit events that the application uses and the parameters that you want to associate with each of the events. The events are specified in an XML file known as the audit event definition file and are used to generate an application-specific, client-side auditing library that sends the application events for auditing to Elasticsearch.
+In order to use Auditing with an application, you must first specify the audit events that the application uses and the parameters that you want to associate with each of the events. The events are specified in an XML file known as the audit event definition file and are used to generate an application-specific, client-side auditing library that sends the application events for auditing to Elasticsearch or the Audit Web Service HTTP REST API.
 
-Elasticsearch receives CAF audit events for a tenant from the client-side library and adds the application audit event to the tenant's index.
+Elasticsearch receives application audit events for a tenant and stores application audit events in the tenant's index.
 
 ## Overview
 
-Auditing is built on Elasticsearch for the messaging and storage of the audit events. Elasticsearch offers high availability, throughput, scalability, and performance to the overall solution. Additionally, Elasticsearch is accessible via RESTful APIs, it offers strong data analytics and monitoring capabilities.
+Auditing is built on Elasticsearch for the messaging and storage of audit events. Elasticsearch offers high availability, throughput, scalability, and performance to the overall solution. Additionally, Elasticsearch is accessible via RESTful APIs, it offers strong data analytics and monitoring capabilities.
+
+The Audit Service client-side library can be configured to send application audit event messages directly to Elasticsearch or via the Audit Web Service HTTP REST API.
 
 ## Audit Service Component Architecture
```

Comment:
A couple of points about this section (although this is my opinion and you may want to seek another):
1. I would show just a single diagram here and within it show the two options (ie. direct and webservice). This will also help minimise the duplication in the sections.
2. I'm not convinced you need a configuration paragraph here for both options. When describing the single diagram, mention it can be configure in one of two (or maybe three if including noopt) ways and refer to documentation link for this (make sure the link works).